<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bookman Old Style</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Server:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: rgba(255,255,255,128);
	border-radius: 5px;
}
QPushButton::hover
{
	background: rgba(255,255,255,192);
	border-radius: 5px;
}
QPushButton::pressed
{
	background: rgba(255,255,255,255);
	border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pw:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: rgba(255,255,255,128);
	border-radius: 5px;
}
QPushButton::hover
{
	background: rgba(255,255,255,192);
	border-radius: 5px;
}
QPushButton::pressed
{
	background: rgba(255,255,255,255);
	border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Demi ITC</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>NeuTalk</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>-20</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/LoginBG.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_server">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_signup">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
text-align: left;</string>
   </property>
   <property name="text">
    <string>No Account?</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_forget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
text-align: left;</string>
   </property>
   <property name="text">
    <string>Forget Pw?</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_1</zorder>
  <zorder>pushButton_connect</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_login</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEdit_pw</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_server</zorder>
  <zorder>pushButton_signup</zorder>
  <zorder>pushButton_forget</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_pw</tabstop>
  <tabstop>pushButton_login</tabstop>
  <tabstop>lineEdit_server</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
